.github/workflows/e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: End to End Tests
+on: 
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest]
+        vscodeVersion:
+          - stable
```

Comment:
should the node version in the matrix?

Reply:
Our testing here is pretty limited at the moment, so I've manually set the node version here to use in the matrix so we can test against different node versions. When we want multiple versions, we can make an update to use the .npmrc value here as well.